### DISCUSSION TITLE
Hidden files on Windows?

### DISCUSSION THREAD

**Discussion by jmigual** [twpayne/chezmoi#2812] [Q&A] [answered]
I'd like to hide my dotfiles on windows. So for example my .gitconfig file has the hidden attribute and it doesn't show on the File Explorer. Does anyone know how to do this with chezmoi? Is it even possible?

Thank you

**Comment by twpayne** [comment, marked as answer]
chezmoi doesn't directly support Windows file attributes, but you should be able to do it with a script, for example, in `run_onchange_after_hide.bat`:

```bat
attrib +h .gitconfig
```
